frontend/packages/dev-console/src/components/topology/redux/reducer.ts

```diff
@@ -0,0 +1,32 @@
+import { Map } from 'immutable';
+import { merge } from 'lodash';
+import { TopologyAction, Actions } from './action';
+import { TOPOLOGGY_FILTERS_LOCAL_STORAGE_KEY, DEFAULT_TOPOLOGY_FILTERS } from './const';
+
+export type State = Map<string, any>;
+
+export const getDefaultTopologyFilters = () => {
+  const filters = localStorage.getItem(TOPOLOGGY_FILTERS_LOCAL_STORAGE_KEY);
+  if (filters) {
+    return merge(DEFAULT_TOPOLOGY_FILTERS, JSON.parse(filters));
```

Comment:
This mutates `DEFAULT_TOPOLOGY_FILTERS` and that is not desired. Any imports of this will have a mutated value. Please make sure you avoid using lodash methods that mutate an object that is not to be mutated (constants, props, state, etc).

> Note: This method mutates object.

Docs: https://lodash.com/docs/4.17.15#merge

A fix, for instance, could be:
```suggestion
    return { ...DEFAULT_TOPOLOGY_FILTERS, ...JSON.parse(filters) };
```